### PR TITLE
[libjepg-turbo] Fix *.pc files

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

File: ports/libjpeg-turbo/portfile.cmake [hunk 1]
```diff
@@ -72,6 +72,13 @@ else(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
     endif()
 endif()
 
+set(_file "${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/libjpeg.pc")
+if(EXISTS "${_file}")
```
Comment (Neumann-A) [inline ports/libjpeg-turbo/portfile.cmake:76]:
```suggestion
if(EXISTS "${_file}" AND VCPKG_TARGET_IS_WINDOWS)
```

**Comment by Neumann-A** [comment]
@JackBoosY Is gdal flaky in CI (x86-windows)?
`NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.28.29333\bin\HostX64\x86\link.EXE"' : return code '0xc0000005'`

**Comment by JackBoosY** [comment]
@Neumann-A Known issue;:  Visual Studio bug.

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
No need to test features.

**Comment by ras0219-msft** [comment]
LGTM, thanks for the fix!

**Pull request merged**